src/js/stores/OfficeStore.js

```diff
@@ -25,25 +26,25 @@ class OfficeStore extends ReduceStore {
     // Exit if we don't have a successful response (since we expect certain variables in a successful response below)
     if (!action.res || !action.res.success) return state;
 
-    let google_civic_election_id;
-    let office_we_vote_id;
+    let googleCivicElectionId;
+    let officeWeVoteId;
     const office = action.res;
-    const new_offices = {};
+    const newOffices = {};
 
     switch (action.type) {
       case "officeRetrieve":
-        new_offices[office.we_vote_id] = office;
+        newOffices[office.weVoteId] = office;
```

Comment:
Since the variable "office.we_vote_id" is coming from the API response, we cannot change the format of this to camel case ("office.weVoteId").

Reply:
Yes, we missed that.  Thanks for catching that.